### PR TITLE
Deprecation of kube ui addon

### PR DESCRIPTION
**Pull request by jonathan-kosgei**
Added a paragraph to point users to kubernetes dashboard, kube-ui is a dead end found that out the hard way.

### CODE REVIEW

**Comment by jonathan-kosgei** [comment]
I signed it!

File: docs/getting-started-guides/ubuntu-calico.md [hunk 1]
```diff
@@ -434,6 +434,12 @@ The Kubernetes UI can be installed using `kubectl` to run the following manifest
 kubectl create -f https://raw.githubusercontent.com/projectcalico/calico-cni/k8s-1.1-docs/samples/kubernetes/master/kube-ui/kube-ui.yaml
 ```
 
+Note: The Kubernetes UI addon is deprecated and has been replaced with Kubernetes dashboard. You can install it by running:
+
+```kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml```
```
Comment (johndmulhausen) [inline docs/getting-started-guides/ubuntu-calico.md]: 439: ```shell
440: (command)
441: ```

Comment (johndmulhausen) [inline docs/getting-started-guides/ubuntu-calico.md]: Then on 440: the command
Then on 441: ```

**Comment by johndmulhausen** [comment]
@jonathan-kosgei ping

**Comment by jonathan-kosgei** [comment]
Hi, this my first time contributing, what exactly do I need to do?

**Comment by jonathan-kosgei** [comment]
Done. Made the changes with the note.

**Comment by jonathan-kosgei** [comment]
https://github.com/jonathan-kosgei/kubernetes.github.io/blob/master/docs/getting-started-guides/ubuntu-calico.md

**Comment by pwittrock** [comment]
@jonathan-kosgei I don't see your changes for John's comments.  Would you update the PR with his suggestions?

**Comment by pwittrock** [comment]
@jonathan-kosgei actually I will just merge and fix the issue

**Pull request merged**